packages/components/demo/example/example.component.html

```diff
@@ -0,0 +1,5 @@
+@if (params.myBooleanParam) {
+  <div class="example-container" [attr.data-variant]="params.variant" (click)="handleClick()">
+    {{ params.myStringParam }}
+  </div>
+}
```

Comment:
nit(blocking): really nice to have the examples of accessing params, although could we also include something that accesses the row value as well? (i.e. `_row.value`)